bindings/netstandard/ElectionGuard/ElectionGuard.Encryption/GeneratedInterop/CiphertextBallot.generated.cs

```diff
@@ -53,6 +53,10 @@ public ElementModQ ManifestHash
                 var status = External.GetManifestHash(
                     Handle, out NativeInterface.ElementModQ.ElementModQHandle value);
                 status.ThrowIfError();
+                if (value.IsInvalid)
```

Comment:
without this we were getting managed objects with null handles. I only fixed it as part of the properties but it probably also exists as part of methods